### PR TITLE
[FIX] website_mail: do not publish internal notes

### PR DESCRIPTION
**Pull request by mart-e**
Description of the issue/feature this PR addresses:
Force the visibility of internal notes to unpublished on website

### CODE REVIEW

File: addons/website_mail/models/mail_message.py [hunk 1]
```diff
@@ -69,3 +69,9 @@ def check_access_rule(self, cr, uid, ids, operation, context=None):
                     raise AccessError(_('The requested operation cannot be completed due to security restrictions. Please contact your system administrator.\n\n(Document type: %s, Operation: %s)') % (self._description, operation))
         return super(MailMessage, self).check_access_rule(cr, uid, ids=ids, operation=operation, context=context)
 
+    @api.model
+    def create(self, vals):
```
Comment (tde-banana-odoo) [inline addons/website_mail/models/mail_message.py]: Nope.

File: addons/website_mail/views/website_mail.xml [hunk 2]
```diff
@@ -69,7 +69,7 @@
             <ul class="media-list o_website_comments">
                 <t t-foreach="object.message_ids" t-as="message">
                     <t t-if="message.website_published or website.env.user.sudo(website.env.user.id).has_group('base.group_website_publisher')">
-                        <li class="media" t-if="message.message_type == 'comment'" t-att-id="'message-%s' % message.id">
+                        <li class="media" t-if="message.message_type != 'comment'" t-att-id="'message-%s' % message.id">
```
Comment (tde-banana-odoo) [inline addons/website_mail/views/website_mail.xml]: Nope.

Comment (tde-banana-odoo) [inline addons/website_mail/views/website_mail.xml]: You should check message.subtype_id.internal == False. But I think portal users cannot read subtype model. To test.

Comment (mart-e) [inline addons/website_mail/views/website_mail.xml]: the conversation does not make sense if I don't include the emails, so probably just exclude the notifiactions

**Pull request merged**